### DISCUSSION TITLE
Difficulty getting MarkdownTextBlock to work.   XMLNS, dependencies etc.

### DISCUSSION THREAD

**Discussion by AlexanderBlackman** [CommunityToolkit/Labs-Windows#571] [Q&A] [answered]
I've downloaded the CommunityToolkit.Labs.WinUI.Controls.MarkdownTextBlock package from your azure feed (0.1.240911-build.1751)

How can I refer to it in C# using statements and Xaml xmlns="using:     ".   Intelisense doesn't do anything.
CommunityToolkit.Labs.WinUI.MarkdownTextBlock"
gets URI/Namespace not found

Also, would it automatically download its dependencies like normal nuget packages?

**Comment by michael-hawker** [comment, marked as answer]
@AlexanderBlackman see the requirements/updates from our 8.1 release notes here: https://github.com/CommunityToolkit/Windows/releases/tag/v8.1.240821 - you probably need to adjust your TFM?

**Reply by AlexanderBlackman** [reply]
Thank you, that fixed it.
